code/identity/src/main/java/com/adobe/marketing/mobile/identity/IdentityExtension.java

```diff
@@ -499,18 +499,8 @@ void handleIdentityRequestReset(@NonNull final Event event) {
                     "handleIdentityRequestReset: Privacy is opt-out, ignoring event.");
             return;
         }
-        mid = null;
-        advertisingIdentifier = null;
-        blob = null;
-        locationHint = null;
-        customerIds = null;
-        pushIdentifier = null;
-
-        if (namedCollection != null) {
-            namedCollection.remove(DataStoreKeys.AID_SYNCED_KEY);
-            namedCollection.remove(DataStoreKeys.PUSH_ENABLED);
-        }
 
+        clearIdentifiers();
         savePersistently(); // clear datastore
```

Comment:
Should we move savePersistently() to clearIdentifiers() as it is part of syncing cleared values to persistence?